k8s/apis/dynamicclient/client.go

```diff
@@ -28,6 +28,9 @@ var dynamicclientlog = logf.Log.WithName("dynamicclient")
 
 func SetClient(config *rest.Config) error {
 	var err error
+	// to do: read this from config
+	config.QPS = 300
+	config.Burst = 300
```

Comment:
what does these two config do?

Reply:
it can improve the qps for dynamic client, default is 10.